<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WPURR72165A" name="Default test" tests="3" failures="1" timestamp="13 Mar 2019 11:18:17 GMT" time="0.927" errors="0">
  <testcase name="@AfterMethod getResult" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at TestSuite.DemoSiteTestCases.getResult(DemoSiteTestCases.java:142)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:709)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@BeforeMethod openBrowser" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- @BeforeMethod openBrowser -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@BeforeMethod openBrowser" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- @BeforeMethod openBrowser -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterTest closeBrowser" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- @AfterTest closeBrowser -->
  <testcase name="registerDetails" time="0.008" classname="TestSuite.DemoSiteTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at DemoPages.Registration.enterDetails(Registration.java:58)
	at TestSuite.DemoSiteTestCases.registerDetails(DemoSiteTestCases.java:116)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- registerDetails -->
  <testcase name="registerDetails" time="0.0" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- registerDetails -->
  <testcase name="alerts" time="0.001" classname="TestSuite.DemoSiteTestCases">
    <skipped/>
  </testcase> <!-- alerts -->
</testsuite> <!-- Default test -->
